persist-cli-tests/src/test/java/io/ballerina/persist/tools/utils/GeneratedSourcesTestUtils.java

```diff
@@ -87,12 +86,12 @@ public static void assertGeneratedSources(String subDir) {
             BuildProject buildProject = BuildProject.load(Paths.get(GENERATED_SOURCES_DIRECTORY).resolve(subDir)
                     .toAbsolutePath());
             Package currentPackage = buildProject.currentPackage();
-            PackageCompilation compilation = currentPackage.getCompilation();
-            if (compilation.diagnosticResult().hasErrors()) {
-                errStream.println("Compilation errors found in generated sources");
-                errStream.println(Arrays.toString(compilation.diagnosticResult().errors().toArray()));
```

Comment:
Don't we need to print the error if there is a compilation issue?

Reply:
Added now [d09fb81](https://github.com/ballerina-platform/persist-tools/pull/240/commits/d09fb81be3c23926275d87bbba2c79c58546db43)